docs/cloud-tutorials/byod-management.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 5
+title: Bringing Your Own Database
+description: Learn how to bring your own PostgreSQL database to DBOS Cloud
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+In this guide, you'll learn how to bring your own PostgreSQL database instance to DBOS Cloud and deploy your applications to it.
+
+:::info
+This feature is currently only available to [DBOS Pro/Custom plan](https://www.dbos.dev/pricing) subscribers.
+:::
+
+### Link Your Database to DBOS Cloud
+
+DBOS Cloud works with any Postgres database as long as it is accessible and properly set up.
+First, DBOS Cloud uses a `dbosadmin` role to manage your application deployments and schema migrations.
+You need to create a `dbosadmin` role in your Postgres instance that has the `LOGIN`, `CREATEDB` and `CREATEROLE` privileges:
+
+```sql
+CREATE ROLE dbosadmin WITH LOGIN CREATEDB CREATEROLE PASSWORD <password>;
+```
+
+Next, link your Postgres instance to DBOS Cloud.
+You must choose a database instance name which is 3 to 16 characters long and contain only lowercase letters, numbers and underscores.
+You will need this database instance name when you register applications.
+Run this command and enter the password for the `dbosadmin` role when prompted.
+
+```shell
+npx dbos-cloud db link <database-instance-name> -H <database-hostname> 
+```
```

Comment:
Question: do you guys think that users will often want to set a port other than 5432? Should we say something like "you must run on port 5432" or, better, allow them to set the port?